webpack/makeConfig.js

```diff
@@ -94,6 +94,7 @@ export default function makeConfig(isDevelopment) {
         new webpack.DefinePlugin({
           'process.env': {
             NODE_ENV: JSON.stringify(isDevelopment ? 'development' : 'production'),
+            DEVTOOLS: isDevelopment,
```

Comment:
Yes thats is right Uglifier is removig if blocks when the condition cannot be modified so if condition is `if (false)` it is completly removed from build